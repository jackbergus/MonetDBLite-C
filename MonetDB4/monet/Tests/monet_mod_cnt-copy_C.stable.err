stderr of test 'monet_mod_cnt-copy_C` in directory 'src/monet` itself:


# 09:47:59 >  
# 09:47:59 >  Mtimeout -timeout 180 Mserver "--config=/ufs/niels/data/monet/Linux/etc/monet.conf" --debug=10   --set monet_prompt= --dbname=Tests_src_monet --set mapi_port=56955 /ufs/niels/data/monet/Linux/share/MonetDB/mapi.mil monet_mod_cnt-copy_C.prologue ; echo  ; echo Over..
# 09:47:59 >  


# 14:24:43 >  
# 14:24:43 >  Mtimeout -timeout 60 MapiClient -u monetdb -P monetdb "--config=/var/tmp/_4.4_PREFIX_MONETDB_GNU_32_d_--enable-debug_/etc/MonetDB.conf" --host=draco --port=46067 < monet_mod_cnt-copy_C.milC
# 14:24:43 >  


# 14:24:45 >  
# 14:24:45 >  Done.
# 14:24:45 >  

