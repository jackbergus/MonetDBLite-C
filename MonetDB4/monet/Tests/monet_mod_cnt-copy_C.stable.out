stdout of test 'monet_mod_cnt-copy_C` in directory 'monet` itself:


# 18:26:37 >  
# 18:26:37 >  Mtimeout -timeout 180 Mserver "--config=/var/tmp/_4.3.13-FixedPetersDDsync_PREFIX_MONET_GNU_32_d_--enable-warning_/etc/monet.conf" --debug=10 --set "monet_mod_path=/var/tmp/_4.3.13-FixedPetersDDsync_PREFIX_MONET_GNU_32_d_--enable-warning_/lib/MonetDB" --set "gdk_dbfarm=/var/tmp/_4.3.13-FixedPetersDDsync_PREFIX_MONET_GNU_32_d_--enable-warning_/var/MonetDB/dbfarm" --set "sql_logdir=/var/tmp/_4.3.13-FixedPetersDDsync_PREFIX_MONET_GNU_32_d_--enable-warning_/var/MonetDB/log" --set mapi_port=53708 --set sql_port=52024 --set monet_prompt= --trace "--dbname=mTests_src_monet" "/var/tmp/_4.3.13-FixedPetersDDsync_PREFIX_MONET_GNU_32_d_--enable-warning_/lib/MonetDB/mapi.mil" monet_mod_cnt-copy_C.prologue ; echo ; echo Over..
# 18:26:37 >  

# Monet Database Server V4.3.13
# Copyright (c) 1993-2004, CWI. All rights reserved.
# compiled for i686-pc-linux-gnu/32bit; dynamically linked.
#---------------------------------#
# BAT:		  monet_mod_cnt	  #
# (oid)		  (int)		  #
#---------------------------------#
[ "_algebra",	  2		  ]
[ "_arith",	  2		  ]
[ "_bat",	  2		  ]
[ "_builtin",	  2		  ]
[ "_constant",	  2	  ]
[ "_mapi",	  2		  ]
[ "_str",	  2		  ]
[ "_streams",	  2		  ]
[ "_sys",	  2		  ]
[ "_trans",	  2		  ]
[ "algebra",	  2		  ]
[ "arith",	  2		  ]
[ "bat",	  2		  ]
[ "builtin",	  2		  ]
[ "constant",	  2	  ]
[ "main",	  2		  ]
[ "mapi",	  2		  ]
[ "str",	  2		  ]
[ "streams",	  2		  ]
[ "sys",	  2		  ]
[ "trans",	  2		  ]
#-------------------------#
# BAT:		  tmp_33  #
# (oid)		  (int)	  #
#-------------------------#
[ "_algebra",	  2	  ]
[ "_arith",	  2	  ]
[ "_bat",	  2	  ]
[ "_builtin",	  2	  ]
[ "_constant",	  2	  ]
[ "_mapi",	  2	  ]
[ "_str",	  2	  ]
[ "_streams",	  2	  ]
[ "_sys",	  2	  ]
[ "_trans",	  2	  ]
[ "algebra",	  2	  ]
[ "arith",	  2	  ]
[ "bat",	  2	  ]
[ "builtin",	  2	  ]
[ "constant",	  2	  ]
[ "main",	  2	  ]
[ "mapi",	  2	  ]
[ "str",	  2	  ]
[ "streams",	  2	  ]
[ "sys",	  2	  ]
[ "trans",	  2	  ]
#-----------------#
# h	diff	  # name
# str	int	  # type
#-----------------#

printf("\nReady.\n");

Ready.
quit();

Over..

# 18:26:38 >  
# 18:26:38 >  Mtimeout -timeout 60 MapiClient "--config=/var/tmp/_4.3.13-FixedPetersDDsync_PREFIX_MONET_GNU_32_d_--enable-warning_/etc/monet.conf" --host=draco --port=53708 < monet_mod_cnt-copy_C.milC
# 18:26:38 >  

#---------------------------------#
# BAT:		  monet_mod_cnt	  #
# (oid)		  (int)		  #
#---------------------------------#
[ "_algebra",	  3		  ]
[ "_arith",	  3		  ]
[ "_bat",	  3		  ]
[ "_builtin",	  3		  ]
[ "_constant",	  3	  ]
[ "_mapi",	  3		  ]
[ "_str",	  3		  ]
[ "_streams",	  3		  ]
[ "_sys",	  3		  ]
[ "_trans",	  3		  ]
[ "algebra",	  3		  ]
[ "arith",	  3		  ]
[ "bat",	  3		  ]
[ "builtin",	  3		  ]
[ "constant",	  3	  ]
[ "main",	  3		  ]
[ "mapi",	  3		  ]
[ "str",	  3		  ]
[ "streams",	  3		  ]
[ "sys",	  3		  ]
[ "trans",	  3		  ]
#-------------------------#
# BAT:		  tmp_63  #
# (oid)		  (int)	  #
#-------------------------#
[ "_algebra",	  3	  ]
[ "_arith",	  3	  ]
[ "_bat",	  3	  ]
[ "_builtin",	  3	  ]
[ "_constant",	  3	  ]
[ "_mapi",	  3	  ]
[ "_str",	  3	  ]
[ "_streams",	  3	  ]
[ "_sys",	  3	  ]
[ "_trans",	  3	  ]
[ "algebra",	  3	  ]
[ "arith",	  3	  ]
[ "bat",	  3	  ]
[ "builtin",	  3	  ]
[ "constant",	  3	  ]
[ "main",	  3	  ]
[ "mapi",	  3	  ]
[ "str",	  3	  ]
[ "streams",	  3	  ]
[ "sys",	  3	  ]
[ "trans",	  3	  ]
#-----------------#
# h	diff	  # name
# str	int	  # type
#-----------------#

# 11:09:24 >  
# 11:09:24 >  Done.
# 11:09:24 >  

